tests/test_images.py

```diff
@@ -49,32 +51,122 @@ def test_exif(pad):
 
 def test_image_attributes(pad):
     for img in (
-        'test.jpg',
+        'test.jpg', # base image, exif-rotated
         'test-sof-last.jpg', # same image but with SOF marker last
-        'test-progressive.jpg', # same image, but with progressive encoding
+        'test-progressive.jpg', # with progressive encoding, rotated in place
     ):
         image = pad.root.attachments.images.get(img)
         assert image is not None
 
-        assert image.width == 512
-        assert image.height == 384
+        assert image.width == 384
+        assert image.height == 512
         assert image.format == 'jpeg'
 
 
-def test_thumbnail_height(builder):
+_SIMILAR_THUMBNAILS = {
```

Comment:
We have these kinds of constants in Lektor. They are lowercase, maybe these should be too?

Reply:
or maybe the others should be uppercase? :) I dunno, it's entirely unimportant, especially in that particular place.

Reply:
@nixjdm what say you?

Reply:
I actually was under the impression that constants in PEP8 was snake case. It turns out I was wrong (https://www.python.org/dev/peps/pep-0008/#constants). I guess I'm more in favor of adhering to the existing style (see https://github.com/lektor/lektor/blob/master/lektor/db.py#L793), but in reality I don't really care that much :sweat_smile:.

Reply:
Oh yay, a style choice between two recommendations in pep8! @xlotlu keep it the way you've got it. Personally I prefer them being all caps to make them stand out. I've got half a mind to change the others that already exist and make some style rules. Not in this PR though.